elifetools/tests/test_parse_jats.py

```diff
@@ -10,7 +8,7 @@
 from ddt import ddt, data, unpack
 from elifetools import parseJATS as parser
 from elifetools import rawJATS as raw_parser
-from elifetools.utils import date_struct, unicode_value
+from elifetools.utils import date_struct
```

Comment:
I take it `unicode_value` was now unused here

Reply:
There were a few modules importing it but not mentioned in the code itself.